### ISSUE TITLE
Request for new command: `@dependabot ungroup`

### ISSUE THREAD

**Issue by cooljeanius** [dependabot/dependabot-core#10614]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature description

Sometimes when dependabot tries to update a set of dependencies as a group for me, it'll cause test failures, and I won't be able to tell which dependency specifically is the one that's causing the failures. In such a situation, I would like to be able to tell dependabot to do a command like `@dependabot ungroup`, which would then separate out the pieces of the grouped pull request into individual smaller pull requests, so that I could then figure out which dependency update specifically is the one leading to failures.

**Comment by nfriedly** [comment]
Yeah, I'd really like this and/or an option to automatically ungroup any time the build fails.
